gsadjust/GSadjust.py

```diff
@@ -651,7 +655,8 @@ def workspace_append(self):
         Append previously-saved workspace to current workspace.
         """
         fname, _ = QtWidgets.QFileDialog.getOpenFileName(
-            None, 'Open File', self.settings.value('current_dir')
+            None, 'Open File', self.settings.value('current_dir'),
+            filter="Workspace files (*.gsa);;P files (*.p)"
```

Comment:
The syntax is a bit weird, but you include each file type containing description, pattern and separated from the next by double-semicolons.

```
<name> *.<ext>;;<name2> *.ext;;
```

Do we still support .p files? I forget if these are the pickle files or something else.